traitsui/testing/tester/ui_wrapper.py

```diff
@@ -60,9 +60,6 @@ class UIWrapper:
 
     Attributes
     ----------
-    target : any
```

Comment:
`target` has since be renamed to `_target` so it is no longer a public attribute.